dowhy/causal_estimator.py

```diff
@@ -686,18 +664,18 @@ def signif_results_tostr(self, signif_results):
 
 
 def estimate_effect(
+    data: pd.DataFrame,
     treatment: Union[str, List[str]],
     outcome: Union[str, List[str]],
-    identified_estimand: IdentifiedEstimand,
     identifier_name: str,
-    method: CausalEstimator,
+    estimator: CausalEstimator,
     control_value: int = 0,
     treatment_value: int = 1,
     test_significance: Optional[bool] = None,
     evaluate_effect_strength: bool = False,
     confidence_intervals: bool = False,
     target_units: str = "ate",
-    effect_modifiers: List[str] = [],
+    effect_modifiers: List[str] = None,
     fit_estimator: bool = True,
     method_params: Optional[Dict] = None,
 ):
```

Comment:
in the docstring below, need to update how estimator is provided. Providing strings is not allowed.